### PR TITLE
Add Evan

### PR DESCRIPTION
**Pull request by evanbiederstedt**
- Not sure if the accented characters work in Markdown

### CODE REVIEW

**Review by manzt — APPROVED** [review]
LGTM, I believe accented chars should be fine

File: _members/evan_biederstedt.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Evan Biederstedt
+name_degree: Evan Biederstedt
+
+photo: evan_biederstedt.jpg
```
Comment (sehilyi) [inline _members/evan_biederstedt.md:5]: This typo might be making the build error.

```suggestion
photo: evan-biederstedt.jpg
```

Comment (evanbiederstedt) [reply _members/evan_biederstedt.md]: sorry, good catch!

File: _members/evan-biederstedt.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: Evan Biederstedt
+name_degree: Evan Biederstedt
+
+photo: evan-biederstedt.jpg
+alt: picture of Evan Biederstedt
```
Comment (sehilyi) [inline _members/evan-biederstedt.md:6]: Could you add a more detailed description? ([Best practices for alt text](https://accessibility.huit.harvard.edu/describe-content-images)) We are currently working on maintaining helpful alt text for all of our headshots!

Example of mine:

> Photograph of a young Asian man with short brown hair and a hint of a smile. He’s wearing a blue sweater in front of a white wall.

**Review by sehilyi — COMMENTED** [review]
I just left one comment about alt text. Otherwise, LGTM!

File: _members/evan-biederstedt.md (hunk 2 above)
Comment (evanbiederstedt) [reply _members/evan-biederstedt.md]: tried again

**Comment by manzt** [comment]
Thanks Sehi!

**Pull request merged**